tests/integration/components/flash-message-test.js

```diff
@@ -111,4 +111,29 @@ if (parseFloat(Ember.VERSION) > 2.0) {
 
     clock.restore();
   });
+
+  test('a custom component can use the close closure action', function(assert) {
+    assert.expect(3);
+
+    let destroyMessage = sinon.spy();
+    this.set('flash', FlashMessage.create({
+      message: 'flash message content',
+      sticky: true,
+      destroyOnClick: false,
```

Comment:
🤔 I'm wondering if there is a way for us to default this to false when yielding/using the close action. It could be confusing if both are enabled.